boards/arm/particle_argon/board.cmake

```diff
@@ -3,9 +3,9 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-board_runner_args(pyocd "--target=nrf52840" "--frequency=4000000")
+# board_runner_args(pyocd "--target=nrf52840" "--frequency=4000000")
```

Comment:
Do we need to disable pyocd runner for some reason?

Reply:
Its for convenience as I you nrfjprog all the time.
Can revert.